sphinx/directives/code.py

```diff
@@ -217,8 +217,8 @@ def read_file(self, filename, location=None):
         except (IOError, OSError):
             raise IOError(_('Include file %r not found or reading it failed') % filename)
         except UnicodeError:
-            raise UnicodeError(_('Encoding %r used for reading included file %r seems to '
-                                 'be wrong, try giving an :encoding: option') %
```

Comment:
Not a bug, all these changes are used to pass testcases in both unix-like OS and Windows.